### DISCUSSION TITLE
A Typescript attempt

### DISCUSSION THREAD

**Discussion by giniedp** [RayTracing/raytracing.github.io#881] [Show and tell]
Just wanted to say thank you for the free available resources. I took some concepts to put my math library into a test. Here is the result:

https://glib.ginie.eu/examples/#/math/simple-raytracer

I was following the Ray Traycing In One Weekand book. It was a great read and was easy to follow. Good work.

**Comment by hollasch** [comment]
Nice! I was thinking it would be fun to implement this in TypeScript.

**Comment by hollasch** [comment]
BTW, you might want to retitle this thread in case anyone else is interested in JS/TS implementations.

**Comment by trevordblack** [comment]
If we ran out of ways to improve the book we could try for interactive figures in three.js

**Reply by hollasch** [reply]
oooooohhh...
